support-frontend/assets/components/paymentMethodSelector/paymentMethodSelectorAccordionRow.tsx

```diff
@@ -0,0 +1,206 @@
+	If content is longer, we'll need to set overflow: auto
+	but only after max-height has been reached.
+	Otherwise, for short content we'll always see a flash
+	of a scrollbar as the row height is transitioning
+	*/
+	max-height: 500px;
+	transition: max-height ${transitions.medium};
+	overflow: hidden;
+	height: auto;
+`;
+
+const collapsedBody = css`
+	max-height: 0;
+	/*
+	TODO:
+	This transition is being ignored as the hidden
+	attribute is applied immediately
+	transition: max-height ${transitions.short};
+	*/
+	overflow: hidden;
+`;
+
+interface ExistingPaymentMethodAccordionRowProps {
+	expanded: boolean;
+	preExistingPaymentMethod: RecentlySignedInExistingPaymentMethod;
+	updateExistingPaymentMethod?: (
+		existingPaymentMethod: RecentlySignedInExistingPaymentMethod,
+	) => void;
+	paymentMethod: PaymentMethod | null;
+	existingPaymentMethod: RecentlySignedInExistingPaymentMethod | undefined;
+}
+
+export function ExistingPaymentMethodAccordionRow({
+	paymentMethod,
+	preExistingPaymentMethod,
+	existingPaymentMethod,
+	updateExistingPaymentMethod,
+	expanded,
+}: ExistingPaymentMethodAccordionRowProps): EmotionJSX.Element {
+	return (
+		<div>
+			<Radio
+				id={`paymentMethod-existing${preExistingPaymentMethod.billingAccountId}`}
+				name="paymentMethod"
+				type="radio"
+				value={preExistingPaymentMethod.paymentType}
+				onChange={() => {
+					updateExistingPaymentMethod?.(preExistingPaymentMethod);
+				}}
+				aria-expanded={expanded}
+				checked={
+					paymentMethod ===
+						mapExistingPaymentMethodToPaymentMethod(preExistingPaymentMethod) &&
+					existingPaymentMethod === preExistingPaymentMethod
+				}
+				arial-labelledby="payment_method"
+				label={
+					<PaymentMethodLabel
+						label={getExistingPaymentMethodLabel(preExistingPaymentMethod)}
+						logo={
+							paymentMethodData[
+								mapExistingPaymentMethodToPaymentMethod(
+									preExistingPaymentMethod,
+								)
+							].icon
+						}
+						isChecked={existingPaymentMethod === preExistingPaymentMethod}
+					/>
+				}
+			/>
+
+			<div css={explainerListContainer}>
+				Used for your{' '}
+				{subscriptionsToExplainerList(
+					preExistingPaymentMethod.subscriptions.map(
+						subscriptionToExplainerPart,
+					),
+				)}
+			</div>
+
+			{/* Accordion Body */}
+			<div css={expanded ? expandedBody : collapsedBody}>
+				<div hidden={!expanded}>
+					i am the expanded body i am the expanded body i am the expanded body i
+					am the expanded body i am the expanded body Lorem ipsum dolor sit
+					amet, consectetur adipisicing elit. Ut expedita autem neque sed in
+					unde impedit molestiae? Asperiores quo accusamus exercitationem ex
+					omnis ipsum voluptatum accusantium culpa! Accusamus, repellat
+					consectetur! Lorem ipsum dolor sit amet consectetur adipisicing elit.
+					Quae, aspernatur et quas vero pariatur magnam dolor velit minus
+					temporibus ad veniam quaerat! Vero molestias quod modi fugit nihil et
+					ad.
```

Comment:
Remove this?

Reply:
This was the temp accordion body for existing payment methods. Has now been updated